definitions/npm/react-redux_v5.x.x/flow_v0.62.0-/react-redux_v5.x.x.js

```diff
@@ -1,7 +1,7 @@
 import type { Dispatch, Store } from "redux";
+import type { ComponentType, ElementConfig } from 'react';
```

Comment:
This does not work, see [here](https://github.com/flowtype/flow-typed/blob/master/CONTRIBUTING.md#dont-import-types-from-other-libdefs).